confgenerator/filter/internal/ast/ast.go

```diff
@@ -148,7 +148,8 @@ then
 			fmt.Fprintf(&out, `record[%s] = {}
 `, p)
 		} else {
-			fmt.Fprintf(&out, `return nil`)
+			fmt.Fprintf(&out, `return nil
```

Comment:
Nit: I think this would be slightly clearer as `"return nil\n"`

Reply:
Fixed.